client/components/modals/EditProfileModal.jsx

```diff
@@ -119,7 +119,7 @@ export class EditProfileModal extends Component {
           id
         })
         .then(() => {
-          swal('Yaayyy!!!', `${username} password updated`, 'success');
+          swal('Success!!!', `${username} password updated`, 'success');
```

Comment:
Seen. _When you edit profile the success modal says Yaay! Can you change this to something more appropriate?_ Resolved